binaries/data/mods/public/gui/prelobby/common/terms/Privacy_Policy.txt

```diff
@@ -41,13 +42,37 @@ The "service" is the 0 A.D. Empires Ascendant Multiplayer Lobby provided by Wild
 
+[font="sans-bold-14"]User rights (GDPR 13):[/font]
+ 1. Contact Wildfire Games, by sending an email to webmaster at wildfiregames dot com (GDPR 13.1.a, GDPR 13.1.b).
+ 2. Right of access to their data (GDPR 15).
+    If you wish to access your data, contact Wildfire Games and provide your pseudonym (GDPR 11.2). We will then need to confirm your identity by asking to log in the service (GDPR 12.6, GDPR 13.2.e).
+    Data may be obtained in a portable, machine-readable format (GDPR 20).
+    The right to obtain a copy of your data does not affect the rights and freedoms of others (GDPR 15.4).
+ 3. Right to rectification of inaccurate data (GDPR 16).
+ 4. Right to erasure of your data if they are not relevant to the development of 0 A.D., if the data was processed unlawfully or illegaly or if the user objects to the processing and has overriding legitimate grounds (GDPR 17).
+ 5. Right to restriction of processing of your data if its accuracy is contested by the user or if the user needs the data in relation to legal claims (GDPR 18).
+ 6. Right to object to the processing of your data concerning them on grounds relating to their particular situation (GDPR 21).
+ 7. Right to lodge a complaint with a supervisory authority (GDPR 13.2.d, GDPR 77).
+
 [font="sans-bold-14"]You understand that:[/font]
-The IP address is the only information processed that could reliably identify your natural person.
-Usernames are chosen freely on this service, can differ from or collide with names of other users on other services by Wildfire Games such as the Forums.
-Wildfire Games ask you to not publish personal data on this service, especially if you are below 16 years of age.
-User submitted content on the service is public, not verifiable to us and does not enable us to identify your natural person.
-Unless your data is stored for a legal obligation such as Abuse, your userdata will become anonymous if you stop using this service.
-
-[font="sans-bold-14"]If you wish to file personal data request in accordance with the General Data Protection Regulation:[/font]
-Please write an email to user-data@wildfiregames.com.
-You will have to authenticate yourself with the service to prevent abuse of personal data requests.
+The IP address is the only information processed that could reliably identify your natural person. (GDPR 4.5)
+Usernames are chosen freely on this service, can differ from or collide with names of same or other users on other services run by Wildfire Games such as the Forums.
+User submitted content on the service is public, not verifiable to us and, according to the Terms of Use, must not enable us to identify your natural person.
+You will have to authenticate yourself with the service to enforce your rights (GDPR 12.6).
+
+[font="sans-bold-14"]Personal information disclosure:[/font]
```

Comment:
My biggest contribution here: IMO according to art 4.1, personal data can be embedded in chat messages and made public. However, this is in 99% of cases done with malicious intentions. So we can comply with Art 28.1 by enforcing a policy where we restrict that as much as possible. If done by another person, that is a ban, and we are friendly with the victim and help them erase the leaked data; if done by the user themselves, this is either stupid and we can only delete the data that we process, not what has been processed by 3rd parties reading the chat, or this is abuse of their rights, we comply (because erasure can't be refused) and we ban.